1.regression_modelling/regression_model_env.yml

```diff
@@ -0,0 +1,22 @@
+name: regression_model_speckles
+channels:
+  - conda-forge
+dependencies:
+- conda-forge::python=3.10
+- conda-forge::jupyter
+- conda-forge::pip
+- conda-forge::opencv
+- conda-forge::tifffile
+- conda-forge::jupyterlab
+- conda-forge::pandas
+- conda-forge::ipykernel
+- conda-forge::nb_conda_kernels
+- conda-forge::scipy
+- conda-forge::numpy
+- conda-forge::scikit-learn
+- conda-forge::seaborn
+- conda-forge::umap-learn
+- conda-forge::matplotlib
+- conda-forge::pyarrow
+- pip:
+  - pycytominer
```

Comment:
consider adding a version here? Might help with reproducibility.

Reply:
This is a bit of a slippery slope for me. Greg defaults more to leaving environments open, which I see, while others in the lab prefer to have more structured versioning. 

I think this repo is in its early stages and I think that this is beyond the scope of this PR. I am happy to turn this into an issue for down the road. Thanks!